README.md

```
@@ -1,10 +1,23 @@
+
+[[sargamicons.com]](https://sargamicons.com/) ♪♪♪ ヽ(ˇ∀ˇ )ゞ
 
 - Built using SVG stroke, providing maximum flexibility on styling.
 - Optimized vector paths and SVGs for better performance.
 - Request a new icon by creating an issue.
 
-[[sargamicons.com]](https://sargamicons.com/) ∠( ᐛ 」∠)_
 
-[![](https://data.jsdelivr.com/v1/package/npm/sargam-icons/badge)](https://www.jsdelivr.com/package/npm/sargam-icons)
+
+[![](https://data.jsdelivr.com/v1/package/npm/sargam-icons/badge)](https://www.jsdelivr.com/package/npm/sargam-icons) ![CodeRabbit Pull Request Reviews](https://img.shields.io/coderabbit/prs/github/planetabhi/sargam-icons?utm_source=oss&utm_medium=github&utm_campaign=planetabhi%2Fsargam-icons&labelColor=171717&color=FF570A&link=https%3A%2F%2Fcoderabbit.ai&label=CodeRabbit+Reviews)
```

markdownlint-cli2 (0.18.1)

Images should have alternate text (alt text)

(MD045, no-alt-text)